components/SelectionInput.tsx

```diff
@@ -15,7 +15,7 @@ const SelectionInput = forwardRef<HTMLInputElement, Props>(
     {
       index,
       id = `option-${index}`,
-      type = "radio",
+      type = "checkbox",
```

Comment:
this does not need to change
```suggestion
      type = "radio",
```